examples/client.c

```diff
@@ -0,0 +1,113 @@
+// Copyright 2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "curl/curl.h"
+#include "curl/header.h"
+
+void usage_print(void) {
+  printf("Usage: client -l <address of the veraison service> -t <token to be verified>\n");
+  printf("       default address for -l option is 127.0.0.1\n");
+}
+
+int main(int argc, char *argv[]) {
+  CURL *curl_handle;
+  CURLcode res;
+  struct curl_header *type;
+  char uri[100] = {0};
+  char token[1024]={0};
+  int opt;
+  char *address="127.0.0.1"; /*default address is localhost*/
+  char port[] = "8080";
+  char *filename;
+
+  if(argc == 1){
+     printf("\n mention the token path using client -t <path> \n");
+     return -1;
+  }
+
+  /* Parse the command line options*/
+  while((opt=getopt(argc,argv,":lth"))!=-1)
+  {
+    switch(opt)
+    {
+      case 'l':
+          if(argv[optind] == NULL) {
+            usage_print();
+            return -1; 
+          }
+          address = argv[optind];
+      break;
+      case 't':
+          if(argv[optind] == NULL) {
+            usage_print();
+            return -1; 
+          } 
+          filename = argv[optind];
+      break;
+      case 'h':
+          usage_print();
+          return -1;
+      case '?':
+          printf("unknown option: %c\n",optopt);
+      break;
+    }
+  }
+
+  sprintf(uri,"http://%s:%s/challenge-response/v1/",address,port);
```

Comment:
We need to use `snprintf` here otherwise, depending on how big `address` and `port` are, `uri` could overflow